packages/gatsby/src/internal-plugins/functions/middleware.ts

```diff
@@ -223,10 +223,10 @@ export function functionMiddlewares(
     setCookies,
     setContext,
     multer().any(),
+    bodyParserMiddlewareWithConfig(`raw`),
     bodyParserMiddlewareWithConfig(`text`),
     bodyParserMiddlewareWithConfig(`urlencoded`),
     bodyParserMiddlewareWithConfig(`json`),
-    bodyParserMiddlewareWithConfig(`raw`),
```

Comment:
This is to make local in line with Gatsby Cloud and generally the order make a bit more sense as the most useful case for being able to define `type` in body parser configuration is to get raw body and we should ensure this has priority over other middlewares